demos/demo_references.bib

```diff
@@ -25,6 +25,14 @@ @InBook{2015:lmscup
                   {http://www1.maths.leeds.ac.uk/~matak/lms-cup2015.pdf}
 }
 
+@Manual{Gidel:2017,
```

Comment:
Hi Anna, this pull request is from kalogirou:master. Only you have the access to make this change!